src/amo/utils/index.js

```diff
@@ -42,20 +42,26 @@ export function getAddonURL(slug: string): string {
   return `/addon/${slug}/`;
 }
 
+// Note that this also excepts an arbitrary number of additional query params
```

Comment:
```suggestion
// Note that this also accepts an arbitrary number of additional query params
```

?

Reply:
Who's the native English speaker here? 🤦